src/org/usfirst/frc/team467/robot/Drive.java

```diff
@@ -169,6 +169,25 @@ public boolean isStopped() {
 		return left.isStopped() && right.isStopped();
 	}
 
+	public void tuneForward(double distanceInFeet, int pidSlot) {
```

Comment:
Should probably add JavaDoc comments to explain the purpose of all of these new methods.